api/src/models/alert-view.ts

```diff
@@ -9,13 +9,16 @@ export const IAlert = z.object({
   severity: z.enum(['info', 'success', 'error', 'warning']),
   data: z.object({}).nullable(),
   record_end_date: z.string().nullable(),
-  status: z.enum(['active', 'expired'])
+  status: z.enum(['active', 'expired']),
+  create_date: z.string()
 });
 
 // Infer types from the schema
 export type IAlert = z.infer<typeof IAlert>;
-export type IAlertCreateObject = Omit<IAlert, 'alert_id' | 'status'>;
-export type IAlertUpdateObject = Omit<IAlert, 'status'>;
+
+export type IAlertUpdateObject = Omit<IAlert, 'status' | 'create_date'>;
+
+export type IAlertCreateObject = Omit<IAlertUpdateObject, 'alert_id'>;
```

Comment:
(Not right or wrong, but something to consider)

Using `Omit` does mean that if the base type changes, it will automatically change these types. Which might be desirable. But, might also be preferable to use `Pick` so that its more explicit which values this type is defining.